docs/software/installing/easybuild.md

```diff
@@ -1,277 +1,292 @@
-# Installing software with EasyBuild
+# EasyBuild
 
 [Lmod_modules]: ../../computing/Lmod_modules.md
 [softwarestacks]: ../../computing/softwarestacks.md
-
-Most software in the central LUMI software stacks is installed through
-[EasyBuild](https://easybuild.io/). The central software stack is kept as
-compact as possible to ease maintenance and to avoid user confusion. E.g.,
-packages for which users request special customisations will never be
-installed in the central software stack. Moreover, due to the technical
-implementation of the software stack on a system the size of LUMI, installing
-software can be disruptive so new software is mostly made available during
-maintenance intervals.
-
-This however does not mean that you may have to wait for weeks before you can get
-the software you need for your project in LUMI. We made it very easy to install
-additional software in your home or project directories (where the latter is the
-better choice as you can then share with the other people in your project) and use
-that software. It requires not much more than loading a module that configure
-EasyBuild for local installations and running EasyBuild with a few recipes that
-can be supplied by the LUMI User Support Team or your national support team or that
-you may write yourself. And this software is then build in exactly the same way as
-it would be in a central installation.
+[helpdesk]: ../../generic/helpdesk.md
+[lumi-g]: ../../computing/systems/lumig.md
+[eap]: ../../eap/index.md
+
+Most software in the central LUMI [software stacks][softwarestacks] is
+installed through [EasyBuild](https://easybuild.io/). The central software
+stack is kept as compact as possible to ease maintenance and to avoid user
+confusion. E.g., packages for which users request special customisations will
+never be installed in the central software stack. Moreover, due to the
+technical implementation of the software stack on a system the size of LUMI,
+installing software can be disruptive so new software is mostly made available
+during maintenance intervals.
+
+This however does not mean that you may have to wait for weeks before you can
+get the software you need for your project on LUMI. We made it very easy to
+install additional software in your home or project directories (where the
+latter is the better choice as you can then share with the other people in your
+project) and use that software. It requires not much more than loading a module
+that configures EasyBuild for local installations and running EasyBuild with a
+few recipes that can be supplied by the [User Support Team][helpdesk] or your
+national support team or that you may write yourself. And this software is then
+built in exactly the same way as it would be in a central installation.
 
```

Comment:
Should it be added (rather/also) here, the information and link to available Easyconfigs?
E.g.

To see the currently available software installation recipes (EasyConfig files) on LUMI, please check the github repository for the available EasyConfig files [here](https://github.com/Lumi-supercomputer/LUMI-SoftwareStack/tree/main/easybuild/easyconfigs)

Reply:
I think that it is fine to have the full list of links to recipe repos towards the end of the page.